test/e2e/installplan_e2e_test.go

```diff
@@ -641,6 +642,176 @@ func TestCreateInstallPlanWithPreExistingCRDOwners(t *testing.T) {
 	})
 }
 
+func TestUpdateInstallPlan(t *testing.T) {
+	defer cleaner.NotifyTestComplete(t, true)
+	t.Run("UpdateSingleExistingCRDOwner", func(t *testing.T) {
```

Comment:
Nice! I think we also want a negative test-case; i.e. CRD upgrade is blocked from a pre-existing owner.